odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2008,6 +2007,8 @@ def _prepare_qcontext(self):
             floor=math.floor,
             ceil=math.ceil,
         )
+        if not self._context.get('no_qcontext_company'):
+            qcontext['res_company'] = self.env.company.sudo()
```

Comment:
Cf my above comment on `no-`-keys. Do you agree that `if self._context.get('set_qcontext_company', True)` is easier to gasp? Side note, we prefer to use `self.env.context` instead of `self._context` but that's just one of our own coding guidelines in metastorm.

```suggestion
        if self.env.context.get('set_qcontext_company', True):
            qcontext['res_company'] = self.env.company.sudo()
```

Reply:
Yep, not not is never a good idea, thought about that but didn't do it, my bad :+1: